application/utils/external_project_parsers/parsers/cheatsheet_extractor.py

```
@@ -92,6 +93,27 @@ def _fallback_summary(markdown: str) -> str:
     return "No summary found."
 
 
+def _get_committed_at(source_path: str) -> str:
+    """Return the ISO 8601 last-commit timestamp for source_path, or 'No timestamp found.'"""
+
+    try:
+        result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)